### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Alexkill667**
- [DEMO LINK](https://Alexkill667.github.io/layout_html-form/)
- [TEST REPORT LINK](https://Alexkill667.github.io/layout_html-form/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,15 @@
...
+}
+
+fieldset {
+  padding-bottom: 0;
+  margin-bottom: 20px;
+}
+
+.none {
+  display: inline-block;
+  /* margin-bottom: 20px; */
```
Comment (qoa11a) [inline src/style.css]: Remove comments.

File: src/index.html [hunk 2]
```diff
@@ -11,7 +11,93 @@
...
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label for="pass">Password: <input type="password" name="pass"></label>
+      </fieldset>
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        Do you love cats?
+        <label class="none">
```
Comment (qoa11a) [inline src/index.html]: It's a bad class name. What does it represent?

**Review by qoa11a — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 2 above)
Comment (Alexkill667) [reply src/index.html]: It's represent that this label is inline-block, because other labels are block

File: src/style.css (hunk 1 above)
Comment (Alexkill667) [reply src/style.css]: Removed

File: src/index.html [hunk 3]
```diff
@@ -11,7 +11,93 @@
...
+            type="text"
+          >
+        </label>
+        <label>Name: <input name="name" type="text"></label>
+        <label>How old are You?
+          <input
+            type="number"
+            name="age"
+            minlength="1"
+            maxlength="3"
```
Comment (OleziO) [inline src/index.html]: Dont need to use it here.

File: src/style.css [hunk 4]
```diff
@@ -1 +1,14 @@
 /* styles go here */
```
Comment (OleziO) [inline src/style.css]: Remove comment.

**Review by OleziO — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 4 above)
Comment (Alexkill667) [reply src/style.css]: Removed

File: src/index.html (hunk 3 above)
Comment (Alexkill667) [reply src/index.html]: Removed

**Review by witflash — APPROVED** [review]
Well done 👍🏻

**Pull request closed without merging**